### PR TITLE
Version 1.0.2

### PR DESCRIPTION
**Pull request by chbatey**
1.0.1 skipped as there is already a manually created 1.0.1 in the redhat catalog

### CODE REVIEW

**Review by patriknw — APPROVED** [review]
LGTM

**Review by edwardcallahan — APPROVED** [review]
Thank you

File: deploy/olm-catalog/akka-cluster-operator/1.0.2/akka-cluster-operator-certified.v1.0.2.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+    - name: Intro to Akka
+      url: https://doc.akka.io/docs/akka/current/guide/introduction.html
+    - name: Intro to Akka Cluster
+      url: https://doc.akka.io/docs/akka/current/common/cluster.html
+    - name: Akka Cluster demo application
+      url: https://github.com/lightbend/akka-java-cluster-openshift
+    - name: Deploying a Lagom application to OpenShift
+      url: https://developer.lightbend.com/guides/openshift-deployment/lagom/index.html
+  version: 1.0.2
+  replaces: akka-cluster-operator-certified.v1.0.0
```
Comment (marcoderama) [inline deploy/olm-catalog/akka-cluster-operator/1.0.2/akka-cluster-operator-certified.v1.0.2.clusterserviceversion.yaml:326]: I know the `replaces` thing is kinda wacky, but should that be replacing v1.0.1?  I don't know.  I could never figure this one out...

File: deploy/olm-catalog/akka-cluster-operator/1.0.2/akka-cluster-operator.v1.0.2.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+    - name: Intro to Akka
+      url: https://doc.akka.io/docs/akka/current/guide/introduction.html
+    - name: Intro to Akka Cluster
+      url: https://doc.akka.io/docs/akka/current/common/cluster.html
+    - name: Akka Cluster demo application
+      url: https://github.com/lightbend/akka-java-cluster-openshift
+    - name: Deploying a Lagom application to OpenShift
+      url: https://developer.lightbend.com/guides/openshift-deployment/lagom/index.html
+  version: 1.0.2
+  replaces: akka-cluster-operator.v1.0.0
```
Comment (marcoderama) [inline deploy/olm-catalog/akka-cluster-operator/1.0.2/akka-cluster-operator.v1.0.2.clusterserviceversion.yaml:320]: ditto

File: deploy/olm-catalog/akka-cluster-operator/1.0.2/akka-cluster-operator-certified.v1.0.2.clusterserviceversion.yaml (hunk 1 above)
Comment (edwardcallahan) [reply deploy/olm-catalog/akka-cluster-operator/1.0.2/akka-cluster-operator-certified.v1.0.2.clusterserviceversion.yaml:326]: you are right. there is no 1.0.0 in the connect registry, so -certified might want to replace 1.0.1 instead.

**Comment by n3ziniuka5** [comment]
Hey @chbatey, I see these changes are already in `master`, but the new version is not in OperatorHub yet and the docker image is also not deployed. Is that intentional?

**Comment by chbatey** [comment]
I’ll look into it soon. The image should have been pushed as a GH. Operator
hub won’t be updated until the PRs there have been merged

**Comment by chbatey** [comment]
Github is having some issues here as this PR has aleady been merged. I've now pushed the v1.0.2 images

**Pull request closed without merging**